_posts/2016-04-14-using-bumbler-to-reduce-runtime-dependencies.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Using Bumbler to reduce runtime dependencies"
+date: 2016-04-14 11:53:00
+categories: ["ruby", "rails", "performance"]
+author: "mauro-oto"
+---
+
+A few weeks ago, I ran into an interesting project called
+ [Bumbler](https://github.com/nevir/Bumbler). Bumbler lets you
+ know which are your Bundler-based project's largest dependencies.
+When you find yourself staring at the screen after running
+ `bundle exec rails c`, you may want to give this tool a try.
+
+<!--more-->
+
+Using it is simple. Add `gem 'bumbler'` to your Gemfile, run `bundle`, and
+ you're good to go by running `bundle exec bumbler`. Running this will show you
+ a progress bar and detail the dependencies which took the longest to load.
```

Comment:
@mauro-oto I'd change this to: 

When you run `bumbler` you will see a progress bar and a detail of the dependencies which took the longest to load.